tests/multiple-tasks/expected_results.json

```diff
@@ -5,55 +5,55 @@
     {
       "status": "pass",
       "name": "square of sum 1",
-      "message": "OK",
+      "message": "assert 1 = square_of_sum 1",
```

Comment:
Sorry if I was unclear. I meant storing it in the `test_code` field. The `message` field should either be `null` or omitted for passing tests.

Reply:
I misunderstood the documentation, thinking that `test_code` was only used for failed cases.